charts/perses/README.md

```diff
@@ -0,0 +1,77 @@
+<!-- Any change to the README file must be must be done on README.md.gotmpl file -->
+
+# perses
+
+Perses helm chart
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.37.2](https://img.shields.io/badge/AppVersion-0.37.2-informational?style=flat-square)
+
+## Installing the Chart
+
+To install the chart with the release name `perses`:
+
+```
+helm repo add perses https://perses.github.io/perses
+helm repo update
+helm install perses perses
+```
+
+## Values
```

Comment:
I'm partially convinced about this way but why not.

Would be cool to have the description so we don't automatically need to look at the file `values.yaml` to know the meaning of the key.